tests/static_map/fetch_insert_test.cu

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) 2022, Jonas Hahnfeld, CERN.
```

Comment:
```suggestion
 * Copyright (c) 2022, NVIDIA CORPORATION.
 * Copyright (c) 2022, Jonas Hahnfeld, CERN.
```

Reply:
I don't see why NVIDIA would have any copyright in this file; I wrote this test from scratch.

Reply:
You're right. I had to check our guidelines on this. Resolved.